### PR TITLE
23 address technical review feedback part 2

### PR DESCRIPTION
**Pull request by brown-jason**
Made a few minor changes from Grace

### CODE REVIEW

File: vignettes/tcplfit2-vignette.Rmd [hunk 1]
```diff
@@ -350,4 +357,15 @@ DT::datatable(res_wide,options = list(scrollX = T))
 
 *The same hitcalling can be done with the full data set, `dat`, as well.*
 
+This output table is the same format as the `res` table in example 3. Users can use the plot code in the chunk that demonstrates the use of `concRespPlot` in example 3 to visualize fits from this output table.
```
Comment (sedavid01) [inline vignettes/tcplfit2-vignette.Rmd:360]: Minor updates to the wording for clarity to the audience:
"The output table resulting from the code in the last code chunk is the same format as the `res` table in example 3.  Thus, users can use the `concRespPlot` function, as done previously in example 3, to plot the results.  The next code chunk demonstrates how to visualize the example 4 fit results."

File: vignettes/tcplfit2-vignette.Rmd [hunk 2]
```diff
@@ -18,22 +18,17 @@ knitr::opts_chunk$set(
...
-and database storage. We wanted to reuse the core concentration-response code for other projects,
-and add extensions to it, which was the origin of the current package `tcplfit2`. The main set of extensions 
-was to include all of the concentration-response models that are contained in the program BMDExpress. 
-These include exponential and power functions in addition to the original Hill, gain-loss and constant models.
+The package tcplfit2 contains the core concentration-response functionality of the package tcpl (The ToxCast Pipeline) built to process all of the ToxCast high-throughput screen (HTS) data at the US EPA. Much of the rest of the code in tcpl is used to do data processing, normalization, and database storage. We wanted to reuse the core concentration-response code for other projects, and add extensions to it, which was the origin of the current package `tcplfit2`. The main set of extensions was to include all of the concentration-response models that are contained in the program [BMDExpress](https://www.sciome.com/bmdexpress/). 
+These include exponential, polynomial, and power functions in addition to the original Hill, gain-loss and constant models.
```
Comment (sedavid01) [inline vignettes/tcplfit2-vignette.Rmd:28]: "exponential, polynomial (1 & 2), and power"

**Review by sedavid01 — CHANGES_REQUESTED** [review]
There are some minor comments to clarify some of the wording added to the vignette.  Additionally, there are two comments from TZ which I do not think are sufficiently addressed but are relatively minor.

- [ ] Add figure legends to the resulting figures for the 4 examples.
- [ ] In the first paragraph for Example 4 we should clarify the statement(s) about tcplLite: "`tcplLite` was deprecated with the updates to `tcpl` and development of `tcplfit2`, since `tcplfit2` allows one to perform curve-fitting and hit-calling independent of a database.  The example below demonstrates how to perform an analogous `tcplLite` analysis with `tcplfit2`.  For additional information, <...>"

Hope these comments are helpful for polishing up the vignette.  Please let me know @brown-jason or @gracezhihuizhao if there are any questions, comments, or concerns.

File: vignettes/tcplfit2-vignette.Rmd [hunk 3]
```diff
@@ -348,6 +348,17 @@ res_wide <- rbindlist(Map(cbind, spid = res$spid, res$V1))
 DT::datatable(res_wide,options = list(scrollX = T))
 ```
 
-*The same hitcalling can be done with the full data set, `dat`, as well.*
+*The same hitcalling can be done with the full data set, `dat`.*
```
Comment (sedavid01) [inline vignettes/tcplfit2-vignette.Rmd:351]: Minor update:
- [ ] "hitcalling can also be done ..."

File: vignettes/tcplfit2-vignette.Rmd [hunk 4]
```diff
@@ -348,6 +348,17 @@ res_wide <- rbindlist(Map(cbind, spid = res$spid, res$V1))
...
 
+The output table resulting from the code in the last code chunk is the same format as the `res` table in example 3. Thus, users can use the `concRespPlot` function, as done previously in example 3, to plot the results. The next code chunk demonstrates how to visualize the example 4 fit results.
```
Comment (sedavid01) [inline vignettes/tcplfit2-vignette.Rmd:353]: Minor updates:
- [ ] "The output table resulting from the previous code chunk is ..."
- [ ] "Thus, one can use ..."

**Review by sedavid01 — COMMENTED** [review]
1. @brown-jason, should we add figure legends to the vignette?
2. Two minor grammatical changes, @gracezhihuizhao do you mind fixing these?

**Comment by brown-jason** [comment]
@sedavid01 plotting in general needs a major overhaul. I think we can add this to future work or even pawn off plotting to tcpl in the futuer where we've had more significant work.

**Comment by sedavid01** [comment]
@brown-jason - Now that I think about it... I originally processed the comment as figure captions NOT figure legends, so that is where I got hung up.  Thank you for your response.  I agree we can move plotting updates to future scope.

I think there is likely good context for the figures in the vignette for now - so we can take or leave updating figure captions in the vignette.  Though if we don't add them this round we definitely should prioritize this with future development of the vignettes in tcplfit2.

Please let me know your thoughts @brown-jason, if you want figure captions or just push all figure updates to future scope.  Thanks!

**Comment by brown-jason** [comment]
@sedavid01 I think we've addressed the technical reviewers at this point so lets push it to future scope

**Comment by sedavid01** [comment]
> @sedavid01 I think we've addressed the technical reviewers at this point so lets push it to future scope

Okay sounds good. Thanks, @brown-jason.

**Review by sedavid01 — APPROVED** [review]
All of the revisions address the technical reviewer comments and last minute text clarification suggestions.  Marking this pull request as "approved".  Thank you again @gracezhihuizhao and @brown-jason for your patience and the changes look great.

**Pull request merged**